packages/nimble-components/src/table-column/text/cell-view.ts

```diff
@@ -0,0 +1,46 @@
+import { observable, volatile } from '@microsoft/fast-element';
+import { DesignSystem } from '@microsoft/fast-foundation';
+import type { TableColumnTextCellRecord, TableColumnTextColumnConfig } from '.';
+import { TableCellView } from '../base/cell-view';
+import { cellViewStyles } from './styles';
+import { cellViewTemplate } from './template';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-table-cell-view-text': TextCellView;
+    }
+}
+
+/**
+ * A cell view for displaying strings
+ */
+export class TextCellView extends TableCellView<
+TableColumnTextCellRecord,
+TableColumnTextColumnConfig
+> {
+    @observable
+    public override cellRecord!: TableColumnTextCellRecord;
+
+    @observable
+    public override columnConfig!: TableColumnTextColumnConfig;
+
+    @observable
+    public isHoveredWithOverflow = false;
+
+    public textSpan!: HTMLElement;
```

Comment:
`textSpan` should be marked as internal

```suggestion
    /** @internal */
    public textSpan!: HTMLElement;
```

Reply:
Updated